config/v1/types_feature.go

```diff
@@ -114,8 +114,6 @@ var FeatureSets = map[FeatureSet]*FeatureGateEnabledDisabled{
 		with("BuildCSIVolumes").                   // sig-build, adkaplan, OCP specific
 		with("NodeSwap").                          // sig-node, ehashman, Kubernetes feature gate
 		with("MachineAPIProviderOpenStack").       // openstack, egarcia (#forum-openstack), OCP specific
-		with("CGroupsV2").                         // sig-node, harche, OCP specific
```

Comment:
In the future, the change should happen by moving this to default and then merging and vendoring down.